l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -681,6 +682,17 @@ def setDettaglioLinea(
             DettaglioLinea.Natura = line.invoice_line_tax_ids[
                 0
             ].kind_id.code
+            if line.invoice_line_tax_ids[0].kind_id.code == 'N2.1' and \
+                    line.invoice_id.partner_id.country_id.code in [
+                'AT', 'BE', 'BG', 'CY', 'HR', 'DK', 'EE', 'FI', 'FR', 'DE', 'GR', 'IE',
+                'IT', 'LV', 'LT', 'LU', 'MT', 'NL', 'PL', 'PT', 'CZ', 'RO', 'SK', 'SI',
+                'ES', 'SE', 'HU'
```

Comment:
E basarci sul campo `intrastat` di `intrastat_base`?

Abbiamo già 3 moduli che potrebbero dipendere da `intrastat_base`: questo, `l10n_it_invoices_data_communication` e `l10n_it_intrastat`

`l10n_it_intrastat` però probabilmente dovrebbe essere ristrutturato per usarne le funzionalità

Reply:
`intrastat_base` l'abbiamo mica appena tolto dalle dipendenze?

Reply:
In effetti sì... https://github.com/OCA/l10n-italy/commit/7d6440c4738d3f5ebc14a1d67b3729e92ee5bdd3